examples/1App_SAS_SodiumLoop/README.md

```diff
@@ -0,0 +1,18 @@
+# 1App_SAS_SodiumLoop
+
+## Purpose
+
+This example provides a demonstration on how to use WATTS to model simple SAS model.
+
+## Code(s)
+ 
+- SAS
+
```

Comment:
I think it's good to let users know that what SAS modules each example uses because each module produces different outputs that require different tools to convert them into csv(or xml) files.